README.md

```diff
@@ -1,7 +1,6 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Basics Greetings Rails app with Webpack: Microverse Project
```

Comment:
- [ ] Please document your project professionally, Your readme file should be descriptive all main headers from the [microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your repo 👍🏿